tensorflow/core/util/util.cc

```diff
@@ -127,24 +127,23 @@ string SliceDebugString(const TensorShape& shape, const int64_t flat) {
   return result;
 }
 
-bool IsMKLEnabled() {
+bool IsOneDNNEnabled() {
+  // For platforms which oneDNN does support, always return false.
 #ifndef INTEL_MKL
   return false;
 #endif  // !INTEL_MKL
-  static absl::once_flag once;
+
+  // When build with --config=mkl option, oneDNN is always enabled.
 #ifdef ENABLE_MKL
-  // Keeping TF_DISABLE_MKL env variable for legacy reasons.
-  static bool oneDNN_disabled = false;
-  absl::call_once(once, [&] {
-    TF_CHECK_OK(ReadBoolFromEnvVar("TF_DISABLE_MKL", false, &oneDNN_disabled));
-    if (oneDNN_disabled) VLOG(2) << "TF-MKL: Disabling oneDNN";
-  });
-  return (!oneDNN_disabled);
-#else
+    return true;
+#endif  // ENABLE_MKL
+
+  // Stock TensorFlow
   // Linux: Turn oneDNN on by default for CPUs with neural network features.
   // Windows: oneDNN is off by default.
-  // No need to guard for other platforms here because INTEL_MKL is only defined
-  // for non-mobile Linux or Windows.
+  // No need to guard for other platforms because INTEL_MKL is only defined
+   // for non-mobile Linux or Windows.
```

Comment:
Nit: Please remove the extra space.
```suggestion
  // for non-mobile Linux or Windows.
```

Reply:
code change done